client/my-sites/checkout/composite-checkout/components/coupon.tsx

```diff
@@ -93,7 +108,7 @@ const animateInRTL = keyframes`
 `;
 
 const CouponWrapper = styled.form`
-	margin: ${ ( props ) => props.marginTop } 0 0;
+	margin: 0;
```

Comment:
This never worked, apparently, or at least has been broken a long time. There is no `marginTop` that I can find anywhere.